impl/src/main/java/org/jboss/cdi/tck/interceptors/tests/contract/interceptorLifeCycle/aroundConstruct/Baz1Interceptor.java

```diff
@@ -17,8 +17,11 @@
 package org.jboss.cdi.tck.interceptors.tests.contract.interceptorLifeCycle.aroundConstruct;
 
 import jakarta.interceptor.AroundConstruct;
+import jakarta.interceptor.Interceptor;
 import jakarta.interceptor.InvocationContext;
 
+@Interceptor
+@BazBinding
```

Comment:
Should have `@Priority` to be enabled? (Not sure, genuine question.)

Reply:
Actually this question applies to a lot of other interceptor classes modified in this PR -- they are often added an interceptor binding annotation, but not `@Priority`, so they are not enabled AFAIK?

Reply:
True - should have priority and I indeed see multiple interceptors having none. This is weird as I'd expect these tests to fail?
Basically, for any interceptor converted from `@Interceptors` to bindings, you also need to add `@Priority` (or other means of enablement but priority is by far the simplest).

Reply:
@starksm64 So checking this PR and this test (`AroundConstructLifeCycleTest`) on Weld, I am seeing failures. Firstly, it fails because several beans do not have bean defining annotation (`@Dependent`) and once that is fixed, it starts crashing on assertions because the interceptors are not enabled and hence never invoked.
This will be similar with other cases I suppose.

How are you executing those tests?